frontend/src/App.svelte

```diff
@@ -71,13 +100,17 @@
           names. You can also look for exact textual phrases (like
           <QueryLink bind:query value={`"creative process"`} />) and prefix
           matches (such as
-          <QueryLink bind:query value={`genom*`} />).
+          <QueryLink bind:query value={`genom*`} />). 
         </p>
         <p>
           Filter by specific attributes like
           <QueryLink bind:query value={`@subject:compsci`} />,
           <QueryLink bind:query value={`@semester:"fall 2022"`} />, and
-          <QueryLink bind:query value={`@level:{graduate}`} />.
+          <QueryLink bind:query value={`@level:{graduate}`} />. 
+        </p>
+        <p>
+          Search for GENED courses specifically by typing "<QueryLink bind:query value={`GENED`} />", 
+          and filter by checking the boxes below.
         </p>
```

Comment:
This copy is unclear, it's not written in active voice, and there are quotes around the QueryLink which aren't used in any of the others. "The boxes below" are mentioned when there are no boxes displayed below as the user reads the text.

here are some alternative options:

- Type `GENED` to search for General Education requirements.
- Looking for Gen Ed courses? Just add `GENED` to your search.
- General Education courses are categorized under `GENED`.

Reply:
writing instructions to guide users is different from writing a detailed report about your feature. discoverability is what you need, make them feel smart. people can figure out an interface if they're given directions, too many directions are distracting though.

Reply:
This is really instructive! I've never thought of UX directions like this; thank you for the note.